tests/phpunit/data/blocks/fixtures/core__gallery__deprecated-1.parsed.json

```diff
@@ -0,0 +1,21 @@
+[
+	{
+		"blockName": "core/gallery",
+		"attrs": {
+			"columns": 2,
+			"align": "wide"
+		},
+		"innerBlocks": [],
+		"innerHTML": "\n<div class=\"wp-block-gallery columns-2 is-cropped alignwide\">\n\t<figure class=\"blocks-gallery-image\">\n\t\t<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==\" alt=\"title\" />\n\t</figure>\n\t<figure class=\"blocks-gallery-image\">\n\t\t<img src=\"data:image/jpeg;base64,/9j/2wBDAAMCAgICAgMCAgIDAwMDBAYEBAQEBAgGBgUGCQgKCgkICQkKDA8MCgsOCwkJDRENDg8QEBEQCgwSExIQEw8QEBD/yQALCAABAAEBAREA/8wABgAQEAX/2gAIAQEAAD8A0s8g/9k=\" alt=\"title\" />\n\t</figure>\n</div>\n",
+		"innerContent": [
+			"\n<div class=\"wp-block-gallery columns-2 is-cropped alignwide\">\n\t<figure class=\"blocks-gallery-image\">\n\t\t<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==\" alt=\"title\" />\n\t</figure>\n\t<figure class=\"blocks-gallery-image\">\n\t\t<img src=\"data:image/jpeg;base64,/9j/2wBDAAMCAgICAgMCAgIDAwMDBAYEBAQEBAgGBgUGCQgKCgkICQkKDA8MCgsOCwkJDRENDg8QEBEQCgwSExIQEw8QEBD/yQALCAABAAEBAREA/8wABgAQEAX/2gAIAQEAAD8A0s8g/9k=\" alt=\"title\" />\n\t</figure>\n</div>\n"
+		]
+	},
+	{
+		"blockName": null,
```

Comment:
Weird, I wonder where these null blocks come from?

Reply:
I also believe this comes from empty lines like Pete mentioned in https://github.com/WordPress/wordpress-develop/pull/4140#issuecomment-1451472711.